<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 85, 127);</string>
  </property>
  <widget class="QLabel" name="L_Tanggal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="T_Struk">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>561</width>
     <height>231</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>15</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>16</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>17</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>18</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>20</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Nama Barang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Harga Barang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jumlah Beli</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jumlah</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total       :</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_Total">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dibayar    :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kembalian :</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_Dibayar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="L_Kembalian">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="L_Tanggal_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Kozuka Gothic Pro B&quot;;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Terimakasih Sudah Belanja Di Toko Kami :)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Kembali">
   <property name="geometry">
    <rect>
     <x>514</x>
     <y>40</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Kembali</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LEName">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>560</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
